### ISSUE TITLE
IWR6843AOPEVM external trigger

### ISSUE THREAD

**Issue by visocopter** [christopherdoer/rio#3]
Hi again,

sorry to bother you, but I saw that you used a IWR6843AOPEVM Rev. F for the dataset recording. How did you synchronise the Radar and IMU via a microcontroller? Particularly, I am interested in how you triggered the IWR6843AOPEVM externally, since there is very little information out there about this.
In the schematics, I could only find the AR_SYNC_IN line, but I am not sure if this is how it is triggered. Did you trigger it via a pin or via USB?

Thanks again.

**Comment by christopherdoer** [comment]
Yeah, that part was a little tricky... Unfortunately, the trigger pin is only accessible via the 60-pin high density connector. We used a 60-pin Samtec connector to access the “AR_SYNC_IN” pin. The radar measurement is then triggered (active high) via a microcontroller board connected to the “AR_SYNC_IN”. In addition, you have to make sure, that the hardware trigger is selected in your radar configuration (line: “frameCfg”) instead of the default software trigger (using the out-of-the-box-demo). 
Hope that helps!

**Comment by visocopter** [comment]
Wow thank you so much! I have one more question though. Did you modify the ti_mmwave_ros_driver? I think by default, it does not publish the noise_db to the pointcloud. So what modifications did you make and are they available somewhere? 
I would love to reproduce your results on my own hardware and contribute more datasets.

Thank you very much again.

**Comment by christopherdoer** [comment]
You are welcome! 
I recorded the datasets with a custom python-based driver which has a slightly different data format. I plan, however, to also support the PCL format of the ti_mmwave_rospkg in the near future, hopefully within the next week. It seems like ti_mmwave_rospkg version 3.3.0 features a PCL format which should work with RIO. Do you mind opening a new issue for that? That would be great! 
I think there is huge potential in radar based navigation, I am looking forward to your results and new datasets!
